### ISSUE TITLE
Download Stop - list index out of range for timestamps not in UTC

### ISSUE THREAD

**Issue by humasak** [Mottl/GetOldTweets3#13]
Hi All,

I tried this program and it works when I tried to get data from a tweet in UK.
The problem I want to get a tweet from Asia countries (timestamps of tweets in the CSV are not in UTC) and after download some tweets, it stops immediately. 

The example command: 

GetOldTweets3 --near "Jakarta Pusat, DKI Jakarta" --within 100mi --since 2018-11-29 --until 2018-11-30

and the result says: "Saved 92600 list index out of range"

Why does this error happen? Is there anything that I need to modify if I download data from not not in UTC country? thank you

**Comment by Mottl** [comment]
`Saved 92600` is the output with the total number of downloaded tweets,
and `list index out of range` is the error message. It will be great if you can provide the last lines of debug.log (`--debug`).

Alternatively, if you are familiar with Python you can try to find out the line number in code which produced `list index out of range`.

I added `--querysearch` to get the smaller amount of tweets and it worked, so I doubt this is related to timezone. More probably something went wrong with HTTP and/or parsing data from Twitter.

```
GetOldTweets3 --querysearch bitcoin --near "Jakarta Pusat, DKI Jakarta" --within 100mi --since 2018-11-29 --until 2018-11-30

Downloading tweets...
Saved 56
```

**Comment by humasak** [comment]
> `Saved 92600` is the output with the total number of downloaded tweets,
> and `list index out of range` is the error message. It will be great if you can provide the last lines of debug.log (`--debug`).
> 
> Alternatively, if you are familiar with Python you can try to find out the line number in code which produced `list index out of range`.
> 
> I added `--querysearch` to get the smaller amount of tweets and it worked, so I doubt this is related to timezone. More probably something went wrong with HTTP and/or parsing data from Twitter.
> 
> ```
> GetOldTweets3 --querysearch bitcoin --near "Jakarta Pusat, DKI Jakarta" --within 100mi --since 2018-11-29 --until 2018-11-30
> 
> Downloading tweets...
> Saved 56
> ```

Hi, thank you for the reply. Just try this command:

GetOldTweets3 --near "Jakarta Pusat, DKI Jakarta" --within 100mi --since 2018-11-28 --until 2018-11-29

I tried several times and the result always "saved 8200 list index out of range". I open the CSV file, it does not get all the tweet on that date (only get a tweet from 2018-11-28 22:07:10;0;0 to 2018-11-28 23:59:59;0;1).

the last line of debug.log = "list index out of range"

Would you please help to solve this error? thanks

**Comment by Mottl** [comment]
The problem was with the incorrect HTML parsing. Fixed in ccc293c.

Please, upgrade to the latest 0.0.9 release.
